bodhi/client/__init__.py

```diff
@@ -16,12 +16,37 @@
 import platform
 import subprocess
 import sys
-import traceback
 
 import click
 
 from bodhi.client import bindings
 
+common_options = [
```

Comment:
Since these options aren't common for all commands, let's rename this to be a little more specific, like ```new_edit_options``` or something.